frontend/Dockerfile

```diff
@@ -2,16 +2,15 @@ FROM node:current-slim as build
 
 WORKDIR /app
 
-COPY package.json package-lock.json tsconfig.json ./
+COPY . .
```

Comment:
True, but `COPY . .` will only copy all files in the frontend folder, right? 
Regardless, 'COPY package.json package-lock.json tsconfig.json ./' doesn't work, and the image builds in a few seconds. If build time becomes a problem we can optimize when we start working on deployment me thinks

Reply:
Yes, all files in the frontend folder, but it was more a question of security than build times, but if it doesn't work, I guess we can create a bug issue for it. Docker images/containers are a vulnerability in modern development due to the 'mystery' behind them and the trust we put in them, so it's nice to be in the careful side of what we put in them (for example, we put secrets in .env files, this might be copied if someone decides to name it something else and .dockerignore doesn't catch it)